workflows/toast_sim_ground.py

```diff
@@ -104,6 +109,21 @@ def parse_config(operators, templates, comm):
         help="Only sample the provided focalplane pixels",
     )
 
+    parser.add_argument(
+        "--fknee",
+        required=False,
+        type=float,
+        help="Override noise model knee frequency [Hz]",
+    )
+
+    parser.add_argument(
+        "--pwv_limit",
```

Comment:
Is it better to have this option at the workflow level, or add it as a trait to the `SimGround` operator?  For example, we could do a [continue in the code here](https://github.com/hpc4cmb/toast/blob/toast3/src/toast/ops/sim_ground.py#L717) if the pwv is too high and avoid creating the observation in the first place, rather than creating it and then deleting it.

Reply:
This was a quick and dirty solution that was needed for the present simulation task.  What we need is some way of identifying the observations to cut before they are distributed to process groups.  Otherwise we run the risk of severe load imbalance.  It also seems that generating a process group without any observations causes the workflow to fail.

The current hack has the benefit of being limited to the ground simulation workflow.  Until we have a proper solution to the problem, I would keep it there.  How about I edit the help string for `--pwv_limit` to reflect this thinking and create an issue to address the full problem at a later date?

Reply:
Ok, it is fine to leave it as-is.  I agree that ideally we would need to have the `SimGround` operator go and instantiate all the weather instances first, prior to distribution among groups.  That is possible, but more work than just adding a trait.